app/views/admin/versions/_event_change_description.html.haml

```diff
@@ -0,0 +1,35 @@
+- if version.event == 'create'
+  = 'submitted new'
+
+- elsif version.event == 'destroy'
+  = 'deleted'
```

Comment:
@differentreality  There is no interface to delete an event.Should I remove this?